tutorials/physics/using_character_body_2d.rst

```diff
@@ -170,29 +180,29 @@ the same collision response:
     # using move_and_collide
     var collision = move_and_collide(velocity * delta)
     if collision:
-        velocity = velocity.slide(collision.normal)
+        velocity = velocity.slide(collision.get_normal())
 
     # using move_and_slide
-    velocity = move_and_slide(velocity)
+    move_and_slide()
 
  .. code-tab:: csharp
 
     // using MoveAndCollide
-    var collision = MoveAndCollide(velocity * delta);
+    var collision = MoveAndCollide(velocity * (float)delta);
```

Comment:
Is this `velocity` variable meant to be the `CharacterBody2D.Velocity` property? In which case it needs to be `Velocity` with uppercase `V`.